mlflow/spacy.py

```diff
@@ -82,36 +84,37 @@ def save_model(
-    :param code_paths: A list of local filesystem paths to Python file dependencies (or directories
-                       containing file dependencies). These files are *prepended* to the system
-                       path when the model is loaded.
-    :param mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
-
-    :param signature: :py:class:`ModelSignature <mlflow.models.ModelSignature>`
-                      describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
-                      The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
-                      from datasets with valid model input (e.g. the training dataset with target
-                      column omitted) and valid model output (e.g. model predictions generated on
-                      the training dataset), for example:
-
-                      .. code-block:: python
-
-                        from mlflow.models import infer_signature
-
-                        train = df.drop_column("target_label")
-                        predictions = ...  # compute model predictions
-                        signature = infer_signature(train, predictions)
-    :param input_example: Input example provides one or several instances of valid
-                          model input. The example can be used as a hint of what data to feed the
-                          model. The given example will be converted to a Pandas DataFrame and then
-                          serialized to json using the Pandas split-oriented format. Bytes are
-                          base64-encoded.
-    :param pip_requirements: {{ pip_requirements }}
-    :param extra_pip_requirements: {{ extra_pip_requirements }}
-    :param metadata: Custom metadata dictionary passed to the model and stored in the MLmodel file.
+    Args:
+        spacy_model: spaCy model to be saved.
+        path: Local path where the model is to be saved.
+        conda_env: {{ conda_env }}
+        code_paths: A list of local filesystem paths to Python file dependencies (or directories
+                    containing file dependencies). These files are *prepended* to the system
+                    path when the model is loaded.
+        mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
+
+        signature: :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                   describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
+                   The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
+                   from datasets with valid model input (e.g. the training dataset with target
+                   column omitted) and valid model output (e.g. model predictions generated on
+                   the training dataset), for example:
+
+                   .. code-block:: python
+
+                     from mlflow.models import infer_signature
```

Comment:
This indent is broken, need a fix

Reply:
Resolved good spot. FYI this doesn't impact rendering.